tests/UI/README.md

```diff
@@ -35,7 +35,12 @@ npm install
 | DB_USER             | Login user of your MySql (default to **`root`**) |
 | DB_NAME             | Name of the MySql database (default to **`prestashop_db`**) |
 | DB_PASSWD           | Password for your MySql (default to **`empty`**) |
+### Maildev parameters
 
+| Parameter           | Description                                          |
+|---------------------|----------------------------------------------------- |
+| SMTP_SERVER             | The smtp server address for maildev (default to **`172.20.0.4`**)|
```

Comment:
This is a weird default value. Any reason ?

Reply:
it's to communicate with the container localhost when the test run on the docker